### DISCUSSION TITLE
Programmatically connect to WIFI

### DISCUSSION THREAD

**Discussion by renanpaiva64** [dotnet/maui#12922] [Q&A]
Hi everyone,

I'd like to know how can I connect programmatically to a WIFI on dotnet MAUI? Is that possible?

**Comment by mphill** [comment]
MAUI exposes all of the underlying OS SDKs via C#. It's THE primary feature of MAUI/Xamarin.

https://stackoverflow.com/questions/36303123/how-to-programmatically-connect-to-a-wifi-network-given-the-ssid-and-password

https://stackoverflow.com/questions/8818290/how-do-i-connect-to-a-specific-wi-fi-network-in-android-programmatically

**Comment by exendahal** [comment]
There’s a package available for connecting Wi-Fi on MAUI. If anyone is interested, feel free to give it a try!
https://github.com/exendahal/maui_wifi_manager
